tests/Integration/goldens/asset/src/V1/gapic_metadata.json

```diff
@@ -0,0 +1,73 @@
+{
+    "schema": "1.0",
+    "comment": "This file maps proto services\/RPCs to the corresponding library clients\/methods",
+    "language": "php",
+    "protoPackage": "google.cloud.asset.v1",
+    "libraryPackage": "Google\\Cloud\\Asset\\V1",
+    "services": {
+        "google.cloud.asset.v1.AssetService": {
```

Comment:
This should NOT be the fully qualified service name, but rather the part after the value in `protoPackage`, ie: simply `"AssetService"` in this case

Reply:
Done.